package.py

```diff
@@ -4,36 +4,76 @@
 # Copyright (c) 2018 The ungoogled-chromium Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
+
 """
 ungoogled-chromium packaging script for Microsoft Windows
 """
 
 import sys
-if sys.version_info.major < 3:
-    raise RuntimeError('Python 3 is required for this script.')
-
 import argparse
-import os
-import platform
-from pathlib import Path
 import shutil
+from pathlib import Path
 
-sys.path.insert(0, str(Path(__file__).resolve().parent / 'ungoogled-chromium' / 'utils'))
```

Comment:
It's necessary to add this path to ensure `filescfg` and `_common` are imported properly.